package.json

```diff
@@ -90,14 +96,11 @@
   ],
   "scripts": {
     "no-prepublish": "npm run build",
```

Comment:
If this is going to be bundled in the npm package, then remove the `no-` part here.

Reply:
Fixed.